forage-android/build.gradle

```diff
@@ -139,6 +139,11 @@ dependencies {
     testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4'
     testImplementation 'com.squareup.moshi:moshi:1.14.0'
     testImplementation 'com.squareup.moshi:moshi-adapters:1.14.0'
+
+    // Use Mockito judiciously (mainly for mocking views)!
+    // Opt for dependency injection and inheritance over Mockito
+    testImplementation 'org.mockito:mockito-core:5.8.0'
```

Comment:
@devinmorgan I have no problem with using mockito, but I wanted to make sure we heard your opinion on it.

Reply:
fd0b28ef34974344ce19f14cb276a44e73dc3927